bin/src/main.rs

```diff
@@ -149,7 +151,20 @@ async fn main() {
     );
 
     #[cfg(feature = "libjournald")]
-    let journald_source = create_source(&config.journald.paths);
+    let (journalctl_source, journald_source) = if config.journald.paths.is_empty() {
+        let journalctl_source = create_journalctl_source()
+            .map(|s| s.map(StrictOrLazyLineBuilder::Strict))
+            .map_err(|e| warn!("Error initializing journalctl source: {}", e));
```

Comment:
if the user didn't manually defined journald paths, I think this shouldn't be an error log.

We could do something like:
```rust
info!("Journalctl source was not initialized");
debug!("Journalctl source Initialization error was: {}", e);
```